clusters/build-clusters/build04/assets/oauth.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  tokenConfig:
+    accessTokenMaxAgeSeconds: 2419200 # 28d
```

Comment:
Could this go into `common_except_app.ci/` folder instead of the cluster-specific one? Looks like b03 has it but b01 and b02 don't - is this something that we could unify?

Reply:
This one is tricky.

Because it is only part of the object is taken into gitops.
The rest is different for each cluster.

`oc-appy` is a bit unpredicable to me if it is for partial objects.
I always make sure I have a copy of the whole object (and no last apply annotation on it) and then apply locally twice the partial objects, and then push it here.
This will ensure it works at the initial state and our applyconfig works fine thereafter.

And this object is also somehow under the control of OSD: IDP set up all lands in the object.
So in short, this is still a semi-manual work and take it into gitops when everything else is configured.

Reply:
:+1: 

Thanks for the explanation!